apis/validator/beacon_committee_subscriptions.yaml

```diff
@@ -1,73 +1,36 @@
-create:
-  post:
-    operationId: "subscribeToBeaconCommitteeSubnet"
-    summary: Subscribe beacon node to commitee attestation subnet
-    description: "If validator is connected to multiple nodes, it should send this request to all of them to insure subnet network stability."
-    tags:
-      - MinimalValidatorApi
-      - Validator
-    requestBody:
-      content:
-        application/json:
-          schema:
-            type: array
-            items:
-              type: object
-              properties:
-                attestation_committee_index:
-                  $ref: '../../beacon-node-oapi.yaml#/components/schemas/Uint64'
-                slot:
-                  allOf:
-                    - $ref: '../../beacon-node-oapi.yaml#/components/schemas/Uint64'
-                    - description: "Should be slot at which validator is assigned to attest"
-                slot_signature:
-                  allOf:
-                    - $ref: '../../beacon-node-oapi.yaml#/components/schemas/Signature'
-                    - description: "Should be valid against given slot and validator pubkey."
-                aggregator_pubkey:
-                  $ref: '../../beacon-node-oapi.yaml#/components/schemas/PublicKey'
-    responses:
-      "200":
-        description: "Slot signature is valid and beacon node is subscribed to given committee attestation subnet."
-      "400":
-        $ref: '../../beacon-node-oapi.yaml#/components/responses/InvalidRequest'
-      "403":
-        description: "Validator is not aggregator"
-      "500":
-        $ref: '../../beacon-node-oapi.yaml#/components/responses/InternalError'
-      "503":
-        $ref: '../../beacon-node-oapi.yaml#/components/responses/CurrentlySyncing'
-
-attestations:
-  get:
-    summary: "Get unaggregated wire attestations"
-    operationId: "getWireAttestations"
-    description: "Endpoint should aggregate all collected attestations and produce AggregateAndProof without proof. Validator running on weak hardware is expected to add proof and publish"
-    tags:
-      - Validator
-    parameters:
-      - name: committee_index
-        in: path
-        required: true
-        description: Index of validator committee
+post:
+  operationId: "subscribeToBeaconCommitteeSubnet"
+  summary: Subscribe beacon node to committee attestation subnet
+  description: "If validator is connected to multiple nodes, it should send this request to all of them to insure subnet network stability."
```

Comment:
That quite annoying thing as it breaks statelessness of api. Basically you have to get aggregated attestation from BN where you subscribed to required subnet otherwise there is a chance that you ask aggreagate from BN that isn't subscribed to your attestation subnet. 

Statelessness in api is quite important as it enables out of box and effective load balancing. Currently you would need some sticky sessions on load balancer and you would need to be aware that you need that.

Example: You have bunch of validators and you setup 5-6 beacon nodes behind load balancer and your validators would subscribe to subnet on one node but request aggregate from another which wasn't subscribed to required subnet and didn't receive those aggregation.